dev/src/dev/debug_qp/viewers.cljs

```diff
@@ -0,0 +1,23 @@
+(ns dev.debug-qp.viewers
+  (:require
+   [clojure.string :as str]
+   [portal.ui.api]))
+
+(defn- format-sql
+  "Placeholder SQL formatter until I figure out how to integrate https://www.npmjs.com/package/sql-formatter."
+  [sql]
+  (reduce
+   (fn [sql sql-keyword]
+     (str/replace sql (re-pattern sql-keyword) (str \newline sql-keyword)))
+   sql
+   ["FROM" "WHERE" "ORDER BY" "LIMIT"]))
+
+(defn- view-sql [sql]
+  [:pre
+   {:style {:color :pink}}
+   (format-sql sql)])
+
+(portal.ui.api/register-viewer!
```

Comment:
Proof-of-concept custom viewer for SQL, next steps are to figure out how to integrate https://www.npmjs.com/package/sql-formatter (which we use on the FE) for better SQL formatting as well as add lots of pretty custom viewers for different parts of the query and MetadataProviders